--- node_modules/react-scripts/config/webpack.config.js	2022-12-04 20:36:43.000000000 +0700
+++ webpack.config.js	2022-12-04 13:18:24.000000000 +0700
@@ -614,6 +614,7 @@
           },
           isEnvProduction
             ? {
+                inject: false,
                 minify: {
                   removeComments: true,
                   collapseWhitespace: true,
@@ -635,7 +636,7 @@
       // https://github.com/facebook/create-react-app/issues/5358
       isEnvProduction &&
         shouldInlineRuntimeChunk &&
-        new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [/runtime-.+[.]js/]),
+        new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [/.(js|css)$/]),
       // Makes some environment variables available in index.html.
       // The public URL is available as %PUBLIC_URL% in index.html, e.g.:
       // <link rel="icon" href="%PUBLIC_URL%/favicon.ico">
